### PR TITLE
Markdown conversion for `ru` - Replace - Performance section :warning: Do not squash :warning: 

### PR DESCRIPTION
**Pull request by SphinxKnight**
Follow-up of #8564
No conversion report as there were no error left.

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning:

File: files/ru/web/performance/critical_rendering_path/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Критические этапы рендеринга
+slug: Web/Performance/Critical_rendering_path
+tags:
+  - Performance
+translation_of: Web/Performance/Critical_rendering_path
+---
+Критические этапы рендеринга (Critical Rendering Path) - это последовательность шагов, которые выполняет браузер, когда преобразуется HTML, CSS и JavaScript в пиксели, которые вы видите на экране. Оптимизация этих шагов улучшает производительность рендера. Эти этапы включают в себя работу с [Document Object Model ](/ru/docs/Web/API/Document_Object_Model)(DOM), [CSS Object Model ](/ru/docs/Web/API/CSS_Object_Model)(CSSOM), деревом рендера (render tree) и компоновкой объектов (layout)
```
Comment (sashasushko) [inline files/ru/web/performance/critical_rendering_path/index.md:8]:
```suggestion
Критические этапы рендеринга (Critical Rendering Path) - это последовательность шагов, которые выполняет браузер, когда преобразуется HTML, CSS и JavaScript в пиксели, которые вы видите на экране. Оптимизация этих шагов улучшает производительность рендера. Эти этапы включают в себя работу с [Document Object Model](/ru/docs/Web/API/Document_Object_Model)(DOM), [CSS Object Model](/ru/docs/Web/API/CSS_Object_Model)(CSSOM), деревом рендера (render tree) и компоновкой объектов (layout)
```

File: files/ru/web/performance/dns-prefetch/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+**В третьих,** хорошей практикой считается использование `dns-prefetch` в паре с `preconnect`. В то время, как `dns-prefetch` срабатывает только при DNS поиске, `preconnect` устанавливает соединение к нужному серверу. Этот процесс включает в себя DNS resolution, установку TCP соединения и установление безопасного соединения ([TLS](/ru/docs/Glossary/TLS) рукопожатие), если оно доступно. Комбинирование этих двух инструкций даёт возможность ещё больше сократить время ожидания для кросс-доменных запросов. Вы можете использовать их вместе таким образом:
+
+```html
+<link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+<link rel="dns-prefetch" href="https://fonts.gstatic.com/">
+```
+
+> **Примечание:**если приложению необходимо установить соединение со множеством доменов, использование preconnect контрпродуктивно. `preconnect` лучше использовать для наиболее критических соединений. Для менее важных достаточно использовать `<link rel="dns-prefetch">` для сокращения времени DNS запроса.
```
Comment (sashasushko) [inline files/ru/web/performance/dns-prefetch/index.md:59]:
```suggestion
> **Примечание:** если приложению необходимо установить соединение со множеством доменов, использование preconnect контрпродуктивно. `preconnect` лучше использовать для наиболее критических соединений. Для менее важных достаточно использовать `<link rel="dns-prefetch">` для сокращения времени DNS запроса.
```

File: files/ru/web/performance/fundamentals/index.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+> **Примечание:**Firefox OS оптимизирована для работы с приложениями, основанными на Web технологиях: [HTML](/ru/docs/Web/HTML), [CSS](/ru/docs/Web/CSS), [JavaScript](/ru/docs/Web/JavaScript) и т.д. За исключением некоторых базовых служб операционной системы, весь код Firefox OS пришёл из Web приложений и движка Gecko. Даже оконный менеджер операционной системы написан на HTML, CSS и JavaScript. В связи с тем, что ядро операционной системы написано на этих технологиях, было критически важно соблюсти производительность этих технологий. В Firefox OS не может быть какого-то "запасного выхода". И это очень полезно для разработчиков, потому что теперь сторонние приложения могут использовать все преимущества оптимизации операционной системы. Не существует какого-то "магического зелья производительности", доступного только для предустановленных приложений.
```
Comment (sashasushko) [inline files/ru/web/performance/fundamentals/index.md:70]:
```suggestion
> **Примечание:** Firefox OS оптимизирована для работы с приложениями, основанными на Web технологиях: [HTML](/ru/docs/Web/HTML), [CSS](/ru/docs/Web/CSS), [JavaScript](/ru/docs/Web/JavaScript) и т.д. За исключением некоторых базовых служб операционной системы, весь код Firefox OS пришёл из Web приложений и движка Gecko. Даже оконный менеджер операционной системы написан на HTML, CSS и JavaScript. В связи с тем, что ядро операционной системы написано на этих технологиях, было критически важно соблюсти производительность этих технологий. В Firefox OS не может быть какого-то "запасного выхода". И это очень полезно для разработчиков, потому что теперь сторонние приложения могут использовать все преимущества оптимизации операционной системы. Не существует какого-то "магического зелья производительности", доступного только для предустановленных приложений.
```

File: files/ru/web/performance/fundamentals/index.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+Звучит очевидно, но для достижения лучшей воспринимаемой скорости загрузки нужно запускать только тот код, который необходим и критичен для запуска приложения. Короткий путь увеличивает скорость.
+
+Web-платформа очень динамична. JavaScript - это динамически типизированный язык, а Web разрешает загружать код, HTML, CSS, изображения и другие ресурсы динамически. Эти функции могут быть использованы для того, чтобы отложить загрузку ресурсов; чтобы сократить критический путь, подвинув загрузку лишнего контента на несколько моментов позже. Такой подход называется "ленивой загрузкой".
+
+Другая проблема, которая может привести к ненужному простою - это ожидание ответа на запросы (например, запрос к базе данных). Чтобы избегать этой проблемы, приложение должно запрашивать данные как можно раньше, ещё во время запуска программы. Тогда к моменту, когда данные понадобятся - они уже будут в системе и приложению не придётся ждать.
+
+> **Примечание:**Для дополнительной информации об ускорении запуска ознакомьтесь с [Optimizing startup performance](/en-US/Apps/Developing/Performance/Optimizing_startup_performance).
```
Comment (sashasushko) [inline files/ru/web/performance/fundamentals/index.md:116]:
```suggestion
> **Примечание:** Для дополнительной информации об ускорении запуска ознакомьтесь с [Optimizing startup performance](/en-US/Apps/Developing/Performance/Optimizing_startup_performance).
```

File: files/ru/web/performance/performance_budgets/index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+---
+Бюджет производительности - это лимит для предотвращения регрессий. Этот бюджет может быть применён к файлам, типам файлов, всем ресурсам приложения, определённым общим показателям (например, [Время до интерактивности](/ru/docs/Glossary/Time_to_interactive)) пользовательским показателям (например, Время до главного элемента) или к пороговым значениям к определённым точкам во времени.
+
+## Зачем нужен бюджет?
+
+Бюджет существует для отражения желаемых вами целей. С бюджетом вы сможете осознанно выстраивать компромиссы между пользовательским опытом и объективным индикаторами эффективности приложения (например, конверсией)
+
+Эти цели можно разбить на категории:
+
+- Временные (например, [Время до интерактивности,](/ru/docs/Glossary/Time_to_interactive) [Первая отрисовка контента](/ru/docs/Glossary/First_contentful_paint)).
```
Comment (sashasushko) [inline files/ru/web/performance/performance_budgets/index.md:14]:
```suggestion
- Временные (например, [Время до интерактивности](/ru/docs/Glossary/Time_to_interactive), [Первая отрисовка контента](/ru/docs/Glossary/First_contentful_paint)).
```

**Review by sashasushko — APPROVED** [review]

**Pull request merged**